server/service/src/sync/api/core.rs

```diff
@@ -20,22 +21,25 @@ pub(crate) struct SyncApiV5 {
     pub(crate) headers: HeaderMap,
 }
 
-fn generate_headers(hardware_id: &str) -> HeaderMap {
+fn generate_headers(hardware_id: &str, sync_version: u32) -> HeaderMap {
     let mut headers = HeaderMap::new();
     headers.insert(
         HeaderName::from_static("msupply-site-uuid"),
         format!("{}", hardware_id).parse().unwrap(),
     );
-    // TODO get version from version system
     headers.insert(
         HeaderName::from_static("app-version"),
-        "1.0".parse().unwrap(),
+        Version::from_package_json().to_string().parse().unwrap(),
```

Comment:
this is not part of the issue, but thought to add it as it's somewhat related

Reply:
Nice!